qemu/tests/vsock_seqpacket_test.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import random
+import time
+import aexpect
```

Comment:
`aexpect` is an avocado-framework module, not a standard module.

Reply:
Updated.